test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,189 @@
+   - stepDelay: specifies the time interval between burst events (*default value: "60"*)
+
+   The workload objects are created using the clusterloader `SteppedLoad` tuningSet. It defines a load that generates a burst of a given size every X seconds, see more [here](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-se). To learn more about clusterloader2 tuningSet, see the following: [tuningSets configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
+   
+   Use the following equation to configure how long an experiment should take in seconds:
+
+   $$t_{exp} = [stepDelay \times \frac{numberOfWorkloadObjects}{burstSize}] + \alpha$$
+
+   Where $\alpha <= 60$ seconds and it corresponds to the amount of time that clusterloader2 takes to cleanup resources after successful execution of a test. 
+   
+   For example, using the default parameters (i.e., burstSize=1, stepDelay=60 seconds, 10 workload objects) the experiment will take 660 seconds at most. Below are some sample parameters:
+
+   | burstSize (#)   | stepDelay (sec) |  numberOfWorkloadObjects  | duration |
+   | --------------- | --------------- | ------------------------- |--------- |
+   |       1         |      60         |           1,440           |  1 day   |
+   |       1         |      5          |          120,960          |  7 days  |
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```bash 
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console 
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```bash 
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console 
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash 
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, set the ``--kubeconfig`` flag to the path of the kubeconfig file used above and use the newly added Kubestellar provider (`--provider=ks`). Then, run the following command to create the workload:
+
+   ```bash
+   cd $CL2_DIR
+   go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
```

Comment:
This displayed command would be more useful to copy-and-paste if the value given for `--kubeconfig` were `${KUBECONFIG:-$HOME/.kube/config}` --- which is exactly what is assumed by the earlier displayed `kubectl` commands that do not specify a kubeconfig.

Reply:
Ok, updated the command as suggested above.